packages/react-native/ReactAndroid/src/main/res/views/textinput/values/styles.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <style name="Theme.ReactNative.TextInput.DefaultBackground" parent="android:Widget.EditText">
+
+    <item name="android:editTextBackground">@drawable/rn_edit_text_material</item>
```

Comment:
Here rn_edit_text_material can also be abc_edit_text_material (https://bit.ly/3k8fX7I). But the ide promts that it is private in a higher version.